NeoML/Python/neoml/Blob.py

```diff
@@ -136,7 +178,21 @@ def load(math_engine, file_path):
 
 
 def asblob(math_engine, data, shape=None, copy=False):
-    """
+    """Organizes the data from a memory buffer into a blob.
+    
+    Parameters
+    ---------
+    math_engine : object
+        The math engine that will perform calculations.
+    data : object
+        A pointer to the data.
+    shape : array of int, default=None
+        The target blob dimensions. If none are specified,
+        a one-element blob will be assumed.
+    copy : bool, default=False
+        Specifies if the data should be copied to another memory block
```

Comment:
Once again, when `copy=False` NeoML will _try_ to use the same memory, but it's not guaranteed (because of GPUs...)